sdk/src/client/bindings/python/examples/10_mint_nft.py

```diff
@@ -1,40 +1,28 @@
-from iota_client import IotaClient, MnemonicSecretManager
+from iota_client import *
 
 # Create an IotaClient instance
 client = IotaClient(nodes = ['https://api.testnet.shimmer.network'])
 
-secret_manager = MnemonicSecretManager("flame fever pig forward exact dash body idea link scrub tennis minute " +
-                                       "surge unaware prosper over waste kitten ceiling human knife arch situate civil")
+secret_manager = MnemonicSecretManager('flame fever pig forward exact dash body idea link scrub tennis minute ' +
+                                       'surge unaware prosper over waste kitten ceiling human knife arch situate civil')
 
 nft_output = client.build_nft_output(
     unlock_conditions=[
-        {
-            "type": 0,
-            "address": {
-                "type": 0,
-                "pubKeyHash": client.bech32_to_hex("rms1qzpf0tzpf8yqej5zyhjl9k3km7y6j0xjnxxh7m2g3jtj2z5grej67sl6l46"),
-            },
-        },
+        UnlockCondition(
+            UnlockConditionType.Address, 
+            Address(AddressType.ED25519, client.bech32_to_hex('rms1qzpf0tzpf8yqej5zyhjl9k3km7y6j0xjnxxh7m2g3jtj2z5grej67sl6l46')),
```

Comment:
I don't know python at all so maybe that's fine, but this is a bit weird tbh. Having the type and the content decorrelated sounds very error prone

Reply:
Error prone? In my python? 😱

Reply:
> Error prone? In my python? 😱

My python? What did I miss? :trollface:

Reply:
> I don't know python at all so maybe that's fine, but this is a bit weird tbh. Having the type and the content decorrelated sounds very error prone

I mean there are always multiple ways to do it. Not sure what the most pythonic way would be, or if there is one 🤔 
We could also just create different classes for different types or something 🤔

Reply:
We could also just make it a string and automatically detect the type, no?

Reply:
After some discussion, I think for now I would just create separate classes like: ED25519Address(bech32_or_hex: str)

Reply:
So you will do later for UnlockCondition what you did for Address?

Reply:
Totally didn't think about it 😅.
Will update all classes. Will need to wait until tonight tho

Reply:
Guess now we could split it up into different files. But I need some sleep now 😆 .

Maybe let's merge this and I can split it up later?